packages/nextjs/app/builders/0xb216270aFB9DfcD611AFAf785cEB38250863F2C9/page.tsx

```diff
@@ -0,0 +1,73 @@
+import Image from "next/image";
+import { GitHubIcon, TwitterIcon } from "./icons";
+import type { NextPage } from "next";
+import { Address } from "~~/components/scaffold-eth";
+
+const socialLinks = [
+  {
+    href: "https://github.com/Rampop01",
+    label: "GitHub Profile",
+    icon: <GitHubIcon />,
+    className:
+      "btn bg-primary hover:bg-primary/90 text-primary-content btn-md normal-case hover:scale-105 hover:-translate-y-1 transition-all duration-300 shadow-lg hover:shadow-primary/30 dark:bg-accent dark:hover:bg-accent/90 dark:text-accent-content",
+  },
+  {
+    href: "https://twitter.com/OmowumiRahmat",
+    label: "Twitter Profile",
+    icon: <TwitterIcon />,
+    className:
+      "btn bg-info hover:bg-info/90 text-info-content btn-md normal-case hover:scale-105 hover:-translate-y-1 transition-all duration-300 shadow-lg hover:shadow-info/30 dark:bg-accent dark:hover:bg-accent/90 dark:text-accent-content",
```

Comment:
Hey! I noticed that both social buttons share many of the same Tailwind classes. To make the code cleaner and easier to maintain, we could move the common classes into the template, and only keep the differences (like background and shadow colors) inside the objects in the array.

That way, we avoid repetition and make it easier to add or update buttons in the future.